### PR TITLE
PEP8 cleanup

### PR DESCRIPTION
**Pull request by echos-code**

### CODE REVIEW

**Comment by globeriz** [comment]
why fix it if it isn't broken ?

File: pgoapi/pgoapi.py [hunk 1]
```diff
@@ -22,28 +22,26 @@
...
-from pgoapi.auth_google import AuthGoogle
-from pgoapi.utilities import parse_api_endpoint
-from pgoapi.exceptions import AuthException, NotLoggedInException, ServerBusyOrOfflineException, NoPlayerPositionSetException, EmptySubrequestChainException, AuthTokenExpiredException, ServerApiEndpointRedirectException, UnexpectedResponseException
-
-from . import protos
 from POGOProtos.Networking.Requests_pb2 import RequestType
 
-logger = logging.getLogger(__name__)
+from . import __title__, __version__, __copyright__
+from .auth_google import AuthGoogle
```
Comment (kvey) [inline pgoapi/pgoapi.py]: All of these relative imports don't make sense to be doing when the package layout is this simple

**Comment by kvey** [comment]
Even with the cleanup, all of the relative imports are questionable at best

**Comment by echos-code** [comment]
Hey guys, it does not seem like you have thoroughly reviewed this pull request. I have update the description to include bullet points.

It also appears that these changes supersede issue #70

**Pull request closed without merging**